blockchains/eth/eth_worker.js

```diff
@@ -29,6 +30,32 @@ class ETHWorker extends BaseWorker {
     this.withdrawalsDecoder = new WithdrawalsDecoder(this.web3, this.web3Wrapper);
   }
 
+  async ethClientRequestWithRetry(...params) {
+    let retries = 0;
+    let retryIntervalMs = 0;
+    while (retries < constants.MAX_RETRIES) {
+      try {
+        const response = await this.ethClient.request(...params);
```

Comment:
should (...params) be spread again, I thought that request would work with an array

Reply:
I think that it would work with an array if it works with a batch of requests. Technically, [<request1>] is a batch, but I'd have to rework how the parameters are given. Currently we have 
```js
('trace_filter', [{
  fromBlock: this.web3Wrapper.parseNumberToHex(fromBlock),
  toBlock: this.web3Wrapper.parseNumberToHex(toBlock)
}])
```